examples/resource_lacework_resource_group_aws/main.tf

```
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+provider "lacework" {}
+
+resource "lacework_resource_group_aws" "aws" {
```

nit: redundant resource name
```suggestion
resource "lacework_resource_group_aws" "example" {
```